examples/chat.py

```diff
@@ -1,15 +1,16 @@
 import asyncio
 import json
+from typing import Set
 
 from aiohttp import web
-from aiohttp.web import Application, Response
+from aiohttp.web import Application, Request, Response, StreamResponse
```

Comment:
Would be great to clean up these imports, the above line (5) is normally how we do these imports, then we can avoid importing all the individual classes.